articles/downloading_data.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Downloading data"
+layout: article
+excerpt: Tutorial on how to download data from synapse with the clients 
+---
+
+## Downloading a file
+
+Every entity in synapse has a unique synapse Id associated with it.  It can be found on every entity page next to `Synapse ID:`, starting with `syn` ending with numbers (ie. `syn00123`)
+
+
+### Entities
```

Comment:
We don't use Entities when we talk about Synapse "entities" there are files, tables and folders ordered into projects.  The fact that they are called entities in the codebase is irrelevant.